gen/calc.py

```diff
@@ -622,13 +622,11 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
             'rexray': {
                 'loglevel': 'info',
                 'modules': {
-                    'default-admin': {
```

Comment:
It appears that the admin module was  moved from RR to libstorage.

https://github.com/codedellemc/rexray/commit/7024e797a3ed8a0846e77c6885735a3a718bb200

I can't see where a similar option exists in `libstorage` though.